Dockerfile.ocp

```diff
@@ -4,7 +4,7 @@ COPY . .
 ENV GOFLAGS="-mod=vendor"
 ENV GO111MODULE=on
 ENV GOPROXY=direct
-RUN if yum install -y prometheus-promu; then export PROMU=/usr/bin/promu; fi && make build
+RUN if yum install -y prometheus-promu; then export PROMU=/usr/bin/promu; fi && make check-git && promu build
```

Comment:
To support both CI and OCP workflows at the same time, you probably need to run `make promu` if the `prometheus-promu` package doesn't exist:

```
RUN if yum install -y prometheus-promu; then export PROMU=/usr/bin/promu; else make promu; fi && make check-git && promu build
```

Reply:
CI and OSBS builds are different branches:

```
RUN make check-git; \
    if yum install -y prometheus-promu; then \
       export PROMU=/usr/bin/promu;
       promu build
     else \
       make build; \
     fi
```